client_plugin/Makefile

```diff
@@ -282,6 +284,11 @@ CLEANESX_SH   := $(DEPLOY_TOOLS) cleanesx
 deploy-esx: clean-esx
 	$(DEPLOY_ESX_SH) "$(ESX)" "$(VIB_BIN)"
 
+test-upgrade:
```

Comment:
need some inline comment that sheds some information about the objective of newly added targets.

next step .. this target should move out from this file ..not now but something to do next.